src/features/presence/PresenceAvatarList.tsx

```diff
@@ -0,0 +1,115 @@
+    interactionType: string;
+    name: string;
+    profileUrl?: string;
+};
+
+export type Props = {
+    avatarAttributes?: React.HTMLAttributes<HTMLDivElement>;
+    className?: string;
+    collaborators: Array<Collaborator>;
+    hideTooltips?: boolean;
+    maxAdditionalCollaborators?: number;
+    maxDisplayedAvatars?: number;
+    onAvatarMouseEnter?: (id: string) => void;
+    onAvatarMouseLeave?: () => void;
+};
+
+function PresenceAvatarList(props: Props, ref: React.Ref<HTMLDivElement>): JSX.Element | null {
+    const {
+        avatarAttributes,
+        className,
+        collaborators,
+        hideTooltips,
+        maxAdditionalCollaborators = 99,
+        maxDisplayedAvatars = 3,
+        onAvatarMouseEnter = noop,
+        onAvatarMouseLeave = noop,
+        ...rest
+    } = props;
+
+    const [activeTooltip, setActiveTooltip] = React.useState<string | null>(null);
+
+    const hideTooltip = (): void => {
+        setActiveTooltip(null);
+        if (onAvatarMouseLeave) {
+            onAvatarMouseLeave();
+        }
+    };
+
+    const showTooltip = (id: string): void => {
+        setActiveTooltip(id);
+        if (onAvatarMouseEnter) {
+            onAvatarMouseEnter(id);
+        }
+    };
+
+    if (!collaborators.length) {
+        return null;
+    }
+
+    return (
+        <div ref={ref} className={classNames('bdl-PresenceAvatarList', className)} {...rest}>
+            {collaborators.slice(0, maxDisplayedAvatars).map(collaborator => {
+                const { id, avatarUrl, name, isActive, interactedAt, interactionType } = collaborator;
+                return (
+                    <Tooltip
+                        key={id}
+                        isShown={!hideTooltips && activeTooltip === id}
+                        position={TooltipPosition.BOTTOM_CENTER}
+                        text={
+                            <PresenceAvatarTooltip
+                                name={name}
+                                interactedAt={interactedAt}
+                                interactionType={interactionType}
+                                isActive={isActive}
+                            />
+                        }
+                    >
+                        <PresenceAvatar
+                            avatarUrl={avatarUrl}
+                            id={id}
+                            isActive={isActive}
+                            name={name}
+                            onBlur={hideTooltip}
+                            onFocus={() => showTooltip(id)}
+                            onMouseEnter={() => showTooltip(id)}
+                            onMouseLeave={hideTooltip}
+                            {...avatarAttributes}
+                        />
+                    </Tooltip>
+                );
+            })}
+
+            {collaborators.length > maxDisplayedAvatars && (
+                <div
+                    className={classNames('bdl-PresenceAvatarList-count', 'avatar')}
```

Comment:
Were any of the renamed classes overridden in our consuming applications?

Reply:
Just did a check and didn't see references